samples/dymaptic.GeoBlazor.Core.Sample.Shared/Pages/Widgets.razor

```diff
@@ -36,6 +36,9 @@
         <div class="form-group">
             <label>Compass: <input type="checkbox" @onchange="@(() => ToggleWidget(nameof(CompassWidget)))"></label>
         </div>
+        <div class="form-group">
```

Comment:
Per discussion, this page is getting crowded, let's revert these changes, keep `LayerList` just on the new page.

Reply:
corrected